packages/icons/src/library/category.js

```diff
@@ -5,7 +5,11 @@ import { SVG, Path } from '@wordpress/primitives';
 
 const category = (
 	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-		<Path d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm.5 16c0 .3-.2.5-.5.5H5c-.3 0-.5-.2-.5-.5V7h15v12zM9 10H7v2h2v-2zm0 4H7v2h2v-2zm4-4h-2v2h2v-2zm4 0h-2v2h2v-2zm-4 4h-2v2h2v-2zm4 0h-2v2h2v-2z" />
+		<Path
+			fill-rule="evenodd"
```

Comment:
`fill-rule` and `clip-rule` should be changed to `fillRule` and `clipRule` respectively.